demo/src/v3/java/com/google/maps/android/utils/demo/MultiLayerDemoActivity.java

```diff
@@ -0,0 +1,172 @@
+/**
+ * DO NOT EDIT THIS FILE.
+ * <p>
+ * This source code was autogenerated from source code within the `demo/src/gms` directory
+ * and is not intended for modifications. If any edits should be made, please do so in the
+ * corresponding file under the `demo/src/gms` directory.
+ */
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.maps.android.utils.demo;
+
+import android.graphics.Color;
+import android.util.Log;
+import android.widget.Toast;
+
+import com.google.android.libraries.maps.CameraUpdateFactory;
+import com.google.android.libraries.maps.model.BitmapDescriptorFactory;
+import com.google.android.libraries.maps.model.LatLng;
+import com.google.android.libraries.maps.model.MarkerOptions;
+import com.google.maps.android.clustering.ClusterManager;
+import com.google.maps.android.collections.GroundOverlayManager;
+import com.google.maps.android.collections.MarkerManager;
+import com.google.maps.android.collections.PolygonManager;
+import com.google.maps.android.collections.PolylineManager;
+import com.google.maps.android.data.geojson.GeoJsonFeature;
+import com.google.maps.android.data.geojson.GeoJsonLayer;
+import com.google.maps.android.data.geojson.GeoJsonLineStringStyle;
+import com.google.maps.android.data.geojson.GeoJsonPolygonStyle;
+import com.google.maps.android.data.kml.KmlLayer;
+import com.google.maps.android.utils.demo.model.MyItem;
+
+import org.json.JSONException;
+import org.xmlpull.v1.XmlPullParserException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+/**
+ * Activity that adds multiple layers on the same map. This helps ensure that layers don't
+ * interfere with one another.
+ */
+public class MultiLayerDemoActivity extends BaseDemoActivity {
+    public final static String TAG = "MultiDemo";
+
+    @Override
+    protected void startDemo(boolean isRestore) {
+        if (!isRestore) {
+            getMap().moveCamera(CameraUpdateFactory.newLatLngZoom(new LatLng(51.403186, -0.126446), 10));
+        }
+
+        // Shared object managers - used to support multiple layer types on the map simultaneously
+        // [START maps_multilayer_demo_init1]
```

Comment:
Would you mind removing the region tags in this file? I actually migrated all snippets displayed on GMP Android docs to this repo: https://github.com/googlemaps/android-samples/tree/main/snippets so these region tags are no longer used.

Reply:
Done in https://github.com/googlemaps/android-maps-utils/pull/811/commits/05720006b0f217dffc2083bb9c844fdda3d3360b

Reply:
Also in https://github.com/googlemaps/android-maps-utils/pull/811/commits/5b3824e4fb7c19061aeeb2c648ba9a11f04071bc for v3 variant